app/config.py

```diff
@@ -299,6 +299,8 @@ class Config(object):
     FUNCTIONAL_TESTS_BROADCAST_SERVICE_NAME = "Functional Tests Broadcast Service"
     FUNCTIONAL_TESTS_BROADCAST_SERVICE_ID = "8e1d56fa-12a8-4d00-bed2-db47180bed0a"
 
+    MAX_THROTTLE_PERIOD = 2 * 60
```

Comment:
Can't we just set this once in the base if it's not different?

Reply:
I've removed the other assignments, I didn't realise this was the base class and the others would inherit from this.